test/test_parse_es.py

```diff
@@ -84,6 +86,134 @@ def test_extract_number_es(self):
             lang='es')), [1, 4, 7, 8, 14, 157])
         self.assertEqual(extract_number("seis punto dos", lang='es'), 6.2)
         self.assertEqual(extract_numbers("un medio", lang='es'), [0.5])
+        self.assertEqual(extract_number("cuarto", lang='es'), 0.25)
+
+        self.assertEqual(extract_number("2.0", lang='es'), 2.0)
+        self.assertEqual(extract_number("1/4", lang='es'), 0.25)
+
+        self.assertEqual(extract_number("dos e media", lang='es'), 2.5)
+        self.assertEqual(extract_number(
+            "catorce e milésima", lang='es'), 14.001)
+
+        self.assertEqual(extract_number("dos punto cero dos", lang='es'), 2.02)
```

Comment:
The parser already handled both, but there weren't any tests for "coma." There are about to be.

Reply:
I stand corrected, and it's a very good thing you noticed this, and yet another reminder why it's so important to test every possible category of input!

The word "coma" works, but the character `,` is not currently parsing.

Reply:
Apparently nobody implemented logic for written decimals. It's just been passing the whole float through the whole parser. This is now its own issue, I reckon.

Reply:
Well, I guess we can live just with "punto" until someone can parse the comma. The point here is the human could say "dos coma cero dos" and Mycroft should understand it as "2.04". As Mycroft is a voice assistant, we won't type "2,02" in the near future.

PS: I'm wondering how french or german people deals with this, as afaik, in those languages decimal separator is comma as well.

Reply:
Yeah, I've put up an issue to add that logic. Happy to tackle it myself, as well, but I've got appointments through the middle of the day, so I won't get to it until this evening at the earliest.

At its most basic, it's just a regex search for a whole word consisting entirely of numbers separated by a comma.

That won't work in the very long term, because somebody might type in `2,02E`, but it would at least solve `2,02`.

I agree that it's a low priority, as the module is mainly for Mycroft and Mycroft is a voice assistant, but it's entirely conceivable that somebody could - right this minute! - try to parse a bunch of text for Mycroft to pronounce, and it would fail on commas-as-decimals.